src/utils/matchName.ts

```diff
@@ -0,0 +1,14 @@
+import { camelize, capitalize } from '@vue/shared'
+
```

Comment:
ooo did not realize vue exported these guys, nice find

Reply:
this is 10x better than my implementation in stubs, I will refactor stubs once we merge this up